cms/djangoapps/contentstore/views/item.py

```diff
@@ -215,6 +225,14 @@ def xblock_handler(request, usage_key_string):
             ):
                 raise PermissionDenied()
 
+            # Libraries have a maximum component limit enforced on them
+            if (isinstance(parent_usage_key, LibraryUsageLocator) and
+                    _is_library_component_limit_reached(parent_usage_key)):
+                return JsonResponse(
+                    {'error': "Library cannot have more than %d components" % settings.MAX_BLOCKS_PER_CONTENT_LIBRARY},
```

Comment:
Since this error can be user-visible in the UI and some users may very well see it even when the system is functioning normally, it would be good to mark it for translation.

```python
'error': _('Libraries cannot have more than {limit} components').format(limit=settings.MAX_BLOCKS_PER_CONTENT_LIBRARY)
```

(and same in the other two occurrences in this PR)

Reply:
Fixed!